cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -178,26 +178,26 @@
        (Pop64 (dst WritableGpr))
 
       ;; Emits a inline stack probe loop.
-      (StackProbeLoop (tmp WritableReg)
-                      (frame_size u32)
-                      (guard_size u32))
+       (StackProbeLoop (tmp WritableReg)
```

Comment:
re-indenting here is fine but could you (i) align the comment above (line 180) with new indent, and (ii) fix the alignment of the comment below (line 185)?

Reply:
This was the fault of an overzealous lisp mode in neovim. I reverted the changes it made and disabled it :)